site-src/blog/2023/0823-welcome-gamma.md

```diff
@@ -0,0 +1,160 @@
+---
+description: >
+  We are excited to announce the v0.8.0 release of Gateway API, where the GAMMA
+  initiative has now reached Experimental status, Gateway and HTTPRoute gain
+  some new capabilities, conformance profiles are supported, and more!
+---
+
+# Gateway API: Welcome, GAMMA!
+
+<small style="position:relative; top:-30px;">
+  :octicons-calendar-24: August 09, 2023 ·
+  :octicons-clock-24: 5 min read
+</small>
+
+We are thrilled to announce the v0.8.0 release of the Gateway API! With this
+release, the work the GAMMA initiative has been doing over the past year has
+reached [Experimental status][status], and the Gateway and HTTPRoute resources
+gain some new capabilities. We look forward to your feedback!
+
+## What is the GAMMA initiative?
+
+Almost from the point where the Gateway API was itself implementable,
+questions have been raised about how it could be used for configuring service
+meshes. The [GAMMA initiative][gamma], started in 2022, is a dedicated
+vendor-neutral workstream within the Gateway API, working on examining how
+best to fit the service mesh into the framework of the Gateway API resources,
+without requiring users of the Gateway API to relearn everything they
+understand about the Gateway API itself.
+
+Over the last year, GAMMA has dug deeply into the challenges and possible
+solutions around using the Gateway API for service mesh. The end result is a
+small number of [enhancement
+proposals](https://gateway-api.sigs.k8s.io/v1beta1/contributing/gep/) that
+subsume many hours of thought and debate, and provide a minimum viable path to
+allow the Gateway API to be used for service mesh. Of particular note:
+
+- [GEP-1324](https://gateway-api.sigs.k8s.io/geps/gep-1324/) provides an
+  overview of the GAMMA goals and some important definitions. This GEP is well
+  worth a read for its discussion of the problem space.
+- [GEP-1426](https://gateway-api.sigs.k8s.io/geps/gep-1426/) defines how to
+  use Gateway API route resources, such as HTTPRoute, to manage traffic within
+  a service mesh.
+- [GEP-1686](https://gateway-api.sigs.k8s.io/geps/gep-1686/) builds on the
+  work of [GEP-1709](https://gateway-api.sigs.k8s.io/geps/gep-1709/) to define
+  a _conformance profile_ for service meshes to be declared conformant with
+  the Gateway API.
+
```

Comment:
Nit: Although I think it's useful to reference the GEPs somewhere, I feel like they may belong closer to the bottom of the blog post in a "further reading" section.